bin/requirements.txt

```diff
@@ -0,0 +1,3 @@
+epispot
+fire
+termcolor
```

Comment:
Remove termcolor as a dependency since `bcolors` is already implemented
```suggestion
```

Reply:
Fixed in commit 95c948507e1f3479508132612195d9f40f7e3dfc